### PR TITLE
set koji-task-id label on source container build,

### PR DESCRIPTION
**Pull request by rcerven**
so it can be later used in koji_import to resolve owner

* OSBS-8380

Signed-off-by: Robert Cerven <rcerven@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [ ] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [ ] Pull request includes link to an osbs-docs PR for user documentation updates

### CODE REVIEW

File: osbs/build/build_requestv2.py [hunk 1]
```diff
@@ -644,6 +644,10 @@ def __init__(self, build_json_store, outer_template=None):
         self.user_params = SourceContainerUserParams(self.build_json_store)
 
     def render(self, validate=True):
+        koji_task_id = self.user_params.koji_task_id.value
+        if koji_task_id is not None:
+            self.set_label('koji-task-id', str(koji_task_id))
```
Comment (athos-ribeiro) [inline osbs/build/build_requestv2.py]: Should we add a test for this?

File: osbs/build/build_requestv2.py [hunk 2]
```diff
@@ -644,6 +644,10 @@ def __init__(self, build_json_store, outer_template=None):
         self.user_params = SourceContainerUserParams(self.build_json_store)
 
     def render(self, validate=True):
+        koji_task_id = self.user_params.koji_task_id.value
```
Comment (MartinBasti) [inline osbs/build/build_requestv2.py]: Can you move this block to superclass and update `BuildRequestV2.render` accordingly?  This is shared functionality between regular image builds and source container image builds, so it should be in common class.

Comment (rcerven) [reply osbs/build/build_requestv2.py]: moved to common

File: osbs/build/build_requestv2.py (hunk 1 above)
Comment (rcerven) [reply osbs/build/build_requestv2.py]: moved to common, is covered by test for regular builds

Comment (MartinBasti) [reply osbs/build/build_requestv2.py]: win-win situation :D

File: osbs/build/build_requestv2.py [hunk 3]
```diff
@@ -86,6 +86,16 @@ def render(self, validate=True):
...
         self.set_data_from_reactor_config(data)
 
+        koji_task_id = self.user_params.koji_task_id.value
+        if koji_task_id is not None:
+            self.set_label('koji-task-id', str(koji_task_id))
+
+            # keep also original task for all manual builds with task
+            # that way when delegated task for autorebuild will be used
+            # we will still keep track of it
+            if self.triggered_after_koji_task is None:
```
Comment (MartinBasti) [inline osbs/build/build_requestv2.py]: This is `BuildRequestV2` specific, attr `self.triggered_after_koji_task` is nto defined in this class

Comment (rcerven) [reply osbs/build/build_requestv2.py]: right

**Comment by athos-ribeiro** [comment]
Is this CI failure related to this PR at all?

**Comment by rcerven** [comment]
> Is this CI failure related to this PR at all?

no, this seems to be in "/usr/lib/python2.7/site-packages/urllib3/util/retry.py"

**Comment by athos-ribeiro** [comment]
It seems there is a new patch introduced in [1] which recently landed in CentOS. Seems to be it.

[1] https://git.centos.org/rpms/python-requests/c/df199af471c63a0a67fbb707b9138ecfc0560b2f?branch=c7

**Comment by rcerven** [comment]
rn updated: set koji-task-id label on source container build, so it can be later used in koji_import to resolve owner

**Pull request merged**